dispatch/resources/messages.py

```diff
@@ -0,0 +1,18 @@
+import dispatch
+
+DISPATCH_ADMIN_HELP_TEXT = """
+Dispatch Admin """+ dispatch.__version__ + """
```

Comment:
No need to supply `dispatch.__version__` here. According to the doc, docopt have an [api to support version](https://github.com/docopt/docopt#api) that we can use